api/libretime_api/settings/__init__.py

```diff
@@ -0,0 +1,46 @@
+# pylint: disable=unused-import
+from os import getenv
+
+from ._internal import (
```

Comment:
How are you differentiating between what should be in `__init__` and `_internal`?

Reply:
Roughly I thought that internal are dev oriented settings that shouldn't change unless you know what you are doing, and __init__ is the entry point, where advanced users might be able to do some tweaking.

I could revert to the single file settings, those are left overs from my previous experiments with multiple settings modules (prod, minimal).